presto-hive/src/main/java/com/facebook/presto/hive/HiveManifestUtils.java

```diff
@@ -149,6 +151,21 @@ public static long getFileSize(Page statisticsPage, int position)
         return partitionMetadata.build();
     }
 
+    public static long getManifestSizeInBytes(ConnectorSession session, PartitionUpdate partitionUpdate, Map<String, String> parameters)
+    {
+        if (isFileRenamingEnabled(session) && partitionUpdate.containsNumberedFileNames()) {
+            if (parameters.containsKey(FILE_NAMES) && parameters.containsKey(FILE_SIZES)) {
+                return parameters.get(FILE_NAMES).length() + parameters.get(FILE_SIZES).length();
+            }
+            List<FileWriteInfo> fileWriteInfos = partitionUpdate.getFileWriteInfos();
+            return compressFileNames(fileWriteInfos.stream().map(FileWriteInfo::getWriteFileName).collect(toImmutableList())).length()
+                    + compressFileSizes(fileWriteInfos.stream().map(FileWriteInfo::getFileSize).filter(Optional::isPresent).map(Optional::get).collect(toImmutableList())).length();
```

Comment:
We don't need this part, this is duplication with `updatePartitionMetadataWithFileNamesAndSizes`